.github/workflows/causal_lm_cpp.yml

```diff
@@ -817,3 +817,21 @@ jobs:
           wget -q https://huggingface.co/datasets/anon8231489123/ShareGPT_Vicuna_unfiltered/resolve/main/ShareGPT_V3_unfiltered_cleaned_split.json
           source ./ov/setupvars.sh
           ./build/samples/cpp/continuous_batching_benchmark/continuous_batching_benchmark -n 5 -m ./TinyLlama-1.1B-Chat-v1.0/ --dataset ./ShareGPT_V3_unfiltered_cleaned_split.json --cache_size 1
+
+  Overall_Status:
```

Comment:
Jobs in this file are conditional and aren't going to run for every PR. Such jobs shouldn't be set as required.

If you are going to make them unconditional, please enable merge_queue trigger.

Reply:
Same for image generation

Reply:
updated